domain-models-runtime/src/main/resources/templates/db_scripts/metadata.ftl

```diff
@@ -22,33 +22,36 @@ CREATE OR REPLACE FUNCTION ${myuniversity}_${mymodule}.set_${table.tableName}_md
 RETURNS TRIGGER AS $$
 DECLARE
   createdDate timestamp WITH TIME ZONE;
-  createdBy text ;
+  createdBy text;
   updatedDate timestamp WITH TIME ZONE;
-  updatedBy text ;
-  injectedId text;
+  updatedBy text;
+
+  dateFormat text := 'YYYY-MM-DD"T"HH24:MI:SS.MSOF';
+  injectedMetadata jsonb;
+
 BEGIN
+  SET TIME ZONE 'UTC';
+
   createdBy = NEW.created_by;
   createdDate = NEW.creation_date;
```

Comment:
Can `createdDate` been removed? If is used one time only where we can directly write `NEW.creation_date`